components/eam/bld/namelist_files/use_cases/1850S_E3SMv1_superfast_ar5-emis.xml

```diff
@@ -176,4 +176,9 @@
 <!-- Turn off ozone dry deposition, as Linoz O3v2 and ozone are not separated for now. Need to turn on ozone dry deposition when interactive tropospheric chemistry is implemented -->
```

Comment:
Is this comment still accurate?

Reply:
If this use case is for E3SMv1/v2, the answer is yes. Otherwise, no.

Reply:
All the use cases on master should be for v3.